exes/ethercat/inc/public/tfc/ec/devices/schneider/atv320/dbus-iface.hpp

```diff
@@ -314,6 +345,17 @@ struct controller {
         token);
   }
 
+  auto run_limitted(bool positive_speedratio) -> bool {
+    using enum motor::errors::err_enum;
+    if (limit_error_ != success) {
```

Comment:
Does this matter?

Reply:
I see now this method is inversed, maybe we should call it `forbid_run_limited` then a true value indicates that running is forbidden.

Reply:
Hmm yes the name is neither obvious or good. How about `is_forbidden`?

Reply:
`is_forbidden` is good.